pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = 'rocm-docs-core'
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
is commitzen supposed to update automatically? I don't understand how it work yet.